.babelrc

```diff
@@ -40,9 +38,8 @@
         "transform-es2015-parameters",
         "transform-es2015-destructuring",
         "transform-es2015-block-scoping",
-        "transform-es2015-typeof-symbol",
-        "transform-object-assign"
+        "transform-es2015-typeof-symbol"
       ]
     }
   }
-}
+}
```

Comment:
oh snap, i just realized `bubleify` is a thing. do you think it's worth switching to?